core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -4543,11 +4543,16 @@ components:
               format: int32
               minimum: 0
               maximum: 65535
+              description: |
+                Tip percentage of the submitted (and rejected) transaction.
             min_tip_percentage_required:
               type: integer
               format: int32
               minimum: 0
               maximum: 65535
+              description: |
+                A lower bound for tip percentage at current mempool state. Anything lower than this will very likely result in a mempool rejection.
+                A value of 0 means there is no tip that can guarantee submission.
```

Comment:
A couple of things:
* I don't think we need to mention the value of 0 / fallback
* When does this occur? If all the mempool is full of max tip transactions? (or the mempool has unrealistic limits) I think we should use 65535 in that case?